Hymns/AppDelegate.swift

```diff
@@ -9,9 +10,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     static let uiTestingFlag = "-UITests"
     #endif
 
+    static let networkMonitor = NWPathMonitor()
+
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
         FirebaseApp.configure()
+        AppDelegate.networkMonitor.start(queue: DispatchQueue.global(qos: .background))
```

Comment:
Interesting. I didn't know you could monitor the network like this.

Could you change SystemUtil#isNetworkAvailable to use this AppDelete thing instead? Then use system util throughout the code instead of this static AppDelegate.

That way we can mock and unit test (See how I do it with HymnsRepository/Spec)